playbook/app/pb_kits/playbook/pb_icon_stat_value/_icon_stat_value.tsx

```diff
@@ -13,7 +11,7 @@ import IconCircle from '../pb_icon_circle/_icon_circle'
 import Title from '../pb_title/_title'
 
 type IconStatValueProps = {
-  aria?: object,
+  aria?: { [key: string]: string; },
```

Comment:
It might be good to determine a consistency with using a `;` or not using one. 
Any recommendations, @nidaqg that you've found? 

Our codebase seems to have about 4 instances on kits where we use a semicolon after the word string. The rest, 79 results, do not.

Reply:
From what I can see, the semi-colon there is not necessary and it's more important to be consistent across a codebase. I'll remove this semi-colon and be mindful of consistency in future conversions! Thanks for pointing it out!